project/__main__.py

```diff
@@ -0,0 +1,801 @@
+'''
+(The features of this codejam project are not to be taken seriously)
+Code Jam 4 submission for High Houses of Adorable Laboratories: HTT
+HTT is used to run a text editor with a built-in onscreen keyboard.
+Go ahead, give it a try! Our tutorial will walk you through how to use it!
+'''
+import asyncio
+import os
+import threading
+import itertools
+import tkinter as tk
+import tkinter.font as tkFont
+from tkinter import filedialog as fd
+from tkinter import messagebox
+from tkinter import ttk
+import json
+import random
+from pathlib import Path
+import wave
+from PIL import Image, ImageTk
+import pyaudio
+
+
+SCRIPT_DIR = Path(__file__).parent
+
+KEY_DESCRIPTION_PATH = SCRIPT_DIR / Path('key_descriptions.json')
+SAVE_DATA_PATH = SCRIPT_DIR / Path('save_data.json')
+WORDS_PATH = SCRIPT_DIR / Path('words.json')
+DEFAULT_SAVE_PATH = SCRIPT_DIR / Path('default_save.json')
+TUTORIALS_PATH = SCRIPT_DIR / Path('tutorials.json')
+IMAGE_PATH = SCRIPT_DIR.parent / Path('img/')
+AUDIO_PATH = SCRIPT_DIR.parent / Path('audio/')
+DEFAULT_DOC_PATH = SCRIPT_DIR.parent / Path('documents/')
+
+IMAGE_PATHS = {'new_icon': IMAGE_PATH / 'new_icon.png',
+               'open_icon': IMAGE_PATH / 'open_icon.png',
+               'save_icon': IMAGE_PATH / 'save_icon.png',
+               'capsule_small': IMAGE_PATH / 'capsule_small.ico',
+               'capsule1': IMAGE_PATH / 'capsule1.png',
+               'capsule2': IMAGE_PATH / 'capsule2.png',
+               'tutorial_smirk': IMAGE_PATH / 'tutorial_smirk.png',
+               'tutorial_neutral': IMAGE_PATH / 'tutorial_neutral.png',
+               'window_icon': IMAGE_PATH / 'window_icon.ico'
+               }
+
+
+audio_player = pyaudio.PyAudio()
+
+AUDIO_CHUNK_SIZE = 1024
+
+
+async def mainloop_coro(root):
+    '''
```

Comment:
triple single quote docstrings? are you trying to kill me?

Reply:
Is that a bad thing? I thought it didn't matter according to pep8, and python defaults to single-quoted when it returns a string?

Reply:
PEP8 doesn't mind if you use single or double quotes for normal strings - but in PEP257 (the docstring PEP) it is stated that you should `For consistency, always use """triple double quotes""" around docstrings.`

Reply:
Ah, thank you!!!